source/publish/push_repo.md

```diff
@@ -96,12 +96,94 @@ government services.
 ### GitHub Pages
 
 With some modification, you can also deploy your site with [GitHub
-Pages](https://pages.github.com/), but we do not support this. To do this, you
+Pages](https://pages.github.com/) [external link], but we do not support this. To do this, you
 could for example use the [`middleman-gh-pages`]
-tool](https://github.com/edgecase/middleman-gh-pages), which we do not
+tool](https://github.com/edgecase/middleman-gh-pages) [external link], which we do not
 support. We also cannot guarantee that all features of the tool will work if
 you deploy your site with GitHub Pages.
 
+### Example: Deploy your documentation to a temporary site on the GOV.UK PaaS
+
+You can deploy your documentation on a temporary site on the GOV.UK PaaS.
+
+This is useful for review of content by non-technical people, for example delivery managers or product owners.
+
+#### Before you start
+
+You must have a GOV.UK PaaS account. This account is called an organisation, or org. Sign up for an org account at [https://www.cloud.service.gov.uk/signup](https://www.cloud.service.gov.uk/signup).
+
+Once you have an org account, you will need a personal account with access to the `gds-tech-writers` org. Ask one of the other tech writers to authorise the creation of your personal account with access to this org.
+
+For more information, refer to the [GOV.UK PaaS documentation on accounts](https://docs.cloud.service.gov.uk/get_started.html#get-an-account).
+
+#### Install the Autopilot plugin
```

Comment:
What is Autopilot? Why do users need it?
It's more of an optional extra that's not really required for a temporary deployment, right?